.github/workflows/ci.yml

```
@@ -19,7 +19,7 @@ jobs:
       fail-fast: true
       max-parallel: 4
       matrix:
-        python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
+        python-version: [3.8, 3.9, '3.10', 3.11, 3.12, 3.13.0-rc.2]
```

The final version is already available
```suggestion
        python-version: [3.8, 3.9, '3.10', 3.11, 3.12, 3.13]
```